bench/utils.py

```diff
@@ -98,12 +98,13 @@ def check_latest_version():
 		return
 
 	if pypi_request.status_code == 200:
-		pypi_version_str = pypi_request.json().get('info').get('version')
-		pypi_version = Version(pypi_version_str)
-		local_version = Version(bench.VERSION)
+		if not bench.VERSION.endswith("dev"):
```

Comment:
How about you add the following even before the imports are made instead. 

```python
if bench.VERSION.endswith("dev"):
    return
```

I don't see the need to change anything else.